src/auth/components/game-img/game-img.vue

```diff
@@ -0,0 +1,108 @@
+import { ThemeStore } from '../../../_common/theme/theme.store';
+<template>
+	<div id="cover-img-credits">
+		<div class="flow-col">
+			<a class="link-unstyled -title" :href="gameUrl">{{ gameTitle }}</a>
+			<a class="link-muted -dev" :href="devUrl"
+				>by <strong>{{ developerName }}</strong></a
```

Comment:
maybe put a new line before the by and after the strong? prettier might format it better